packages/oft-sand/contracts/sand/WithAdmin.sol

```diff
@@ -11,7 +13,9 @@ contract WithAdmin {
     event AdminChanged(address indexed oldAdmin, address indexed newAdmin);
 
     modifier onlyAdmin() {
-        require(msg.sender == _admin, "ADMIN_ONLY");
+        if (msg.sender != _admin) {
```

Comment:
msg.sender - is this correct?

Reply:
Had this discussion with Maxime in the past. We tried upgrading it but couldn't because of some issue during upgrade, plus these two additional points
1. Sandbox is one with admin role and admin wouldn't be doing gasless transactions.
2. To use _msgSender() here, we will have to upgrade all the other contract inheriting it ( Sand, PolygonSand)

Reply:
Resolved: we could use _msgSender() here for new networks but this is not needed as admin will not call the contract via a relayer